saleor/warehouse/availability.py

```diff
@@ -50,15 +51,39 @@ def get_available_quantity(variant: "ProductVariant", country_code: str) -> int:
 
 
 def get_available_quantity_for_customer(
-    variant: "ProductVariant", country_code: str
+    variant: "ProductVariant", country_code: str = None
 ) -> int:
-    """Return maximum checkout line quantity."""
-    stocks = Stock.objects.get_variant_stocks_for_country(country_code, variant)
+    """Return maximum checkout line quantity.
+
+    Returns maximum checkout quantity for the given variant and country code.
+    If country code is missing return maximum checkout quantity from shipping zones.
```

Comment:
Description:

```
Returns maximum checkout quantity for the given variant and country code. If country code is provided, the function returns the exact variant quantity available in warehouses operating in shipping zones containing this country. Otherwise, it returns the maximum quantity from all warehouses.

The returned value is limited by `MAX_CHECKOUT_LINE_QUANTITY` setting to limit the quantity of a variant that can be added in one checkout line.
```

Reply:
A bit long but I think it's important to give a precise description here.